packages/frontend/src/components/Nav.tsx

```diff
@@ -28,7 +28,7 @@ const useStyles = makeStyles((theme) =>
       zIndex: 30,
       //background: theme.palette.background.default,
       borderBottom: `1px solid ${theme.palette.background.stone}`,
-      backdropFilter: 'blur(30px)',
+      backgroundColor: 'black',
```

Comment:
Thanks for the PR. Instead of removing the backdrop altogether, Can we apply different yet subtle background only for Mozilla? That way we retain the original style for other browsers!

https://stackoverflow.com/questions/56593433/how-to-write-mozilla-specific-css-using-withstyles

Reply:
We can. Per the [backdrop filter](https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter#browser_compatibility) page you linked, Mozzila and Internet Explorer don't support backdrop filter so I suspect we would want to special case styles for both those browsers.

I don't have visibility into the front-end designs for Squeeth so I don't know what styles this nav should conform to. I'll close this PR and let maintainers tackle https://github.com/onbjerg/foundry-book/pull/79 in whichever way aligns with front-end design styles.